MaterialLib/MPL/Properties/CapillaryPressureSaturation/CapillaryPressureVanGenuchten.h

```diff
@@ -0,0 +1,147 @@
+/**
+ * \file
+ * \copyright
+ * Copyright (c) 2012-2020, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ * Created on March 20, 2020, 9:59 AM
+ */
+
+#pragma once
+
+#include "SaturationVanGenuchten.h"
+
+namespace MaterialPropertyLib
+{
+ /// \copydoc MaterialPropertyLib::SaturationVanGenuchten
+class CapillaryPressureVanGenuchten : public SaturationVanGenuchten
```

Comment:
derive from property

Reply:
class hierarchies represent a IsA relation. It would mean here, that "capillary pressure" is "saturation", which is not good at all.

Reply:
Changed the hierarch.